### PR TITLE
Support for OneTouch UltraMini

### PR DESCRIPTION
**Pull request by gniezen**
We've had the UltraMini driver in the codebase for a while now, but it has always been disabled since it didn't have proper error handling and didn't adhere to the UTC bootstrapping time format.

This PR renames `OneTouchMini` as `OneTouchUltraMini` and adds:

- TZOUtil across-the-the-board time
- units from meter, not hard-coded
- BGM checklist
- annotating HI/LOs

### CODE REVIEW

**Comment by gniezen** [comment]
[OneTouch Ultra 2 and OneTouch Mini](https://trello.com/c/drBwiv9K/198-onetouch-ultra-2-and-onetouch-mini)

**Comment by DorianScholz** [comment]
I only had time for a quick look so far.
I would propose a few minor changes which I've pushed in my branch:
https://github.com/DorianScholz/chrome-uploader/commits/dorianscholz/ultramini-cleanup-review

Should I file them as a PR against this branch?

I'll take a more in-depth look in the next days.

File: lib/drivers/onetouch/oneTouchUltraMini.js [hunk 1]
```diff
@@ -511,11 +564,17 @@ module.exports = function (config) {
...
-        _.assign(data, obj);
-        debug('getConfigInfo', data);
-        cb(err, data);
+        if (err) {
+          return cb(err, null);
+        }
+
+        getUnitSettings(obj, function(err, obj2) {
+          progress(100);
+          data.getConfigInfo = true;
```
Comment (DorianScholz) [inline lib/drivers/onetouch/oneTouchUltraMini.js]: shouldn't the value of "err" be checked before setting this flag?

**Comment by gniezen** [comment]
I have addressed the inline comment in a new commit. The change in your branch look good! Yes, please create a new PR against this branch, then I can approve those changes.

**Comment by DorianScholz** [comment]
Your last commit looks good.
I opened a PR for my proposed changes.

**Comment by gniezen** [comment]
@pazaan Can you also approve this one - it has also been reviewed by Dorian already.

**Pull request merged**